modules/oadp-configuring-client-burst-qps.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-aws.adoc
+
+
+:_mod-docs-content-type: PROCEDURE
+[id="oadp-configuring-client-burst-qps_{context}"]
+= Configuring the DPA with client burst and queries per second (QPS) settings
+
+You can set the burst and QPS values of the `velero` server by configuring the Data Protection Application (DPA) with the burst and QPS values. You can use the `dpa.configuration.velero.client-burst` and `dpa.configuration.velero.client-qps` fields of the DPA to set the burst and QPS values.
```

Comment:
Do you think we should add a QPS definition to an introductory paragraph, something like

```
The queries per second (QPS) setting determines the number of client requests or queries that are sent to the Velero server. While the burst value determines how many requests from the client can be stored for processing.
```

Then we could shorten the title to 

```
= Configuring the DPA with client burst and QPS settings
```

Reply:
Done. thank you.